bootstrap-core/src/main/java/de/agilecoders/wicket/core/settings/BootstrapSettings.java

```diff
@@ -57,8 +57,8 @@ public BootstrapSettings() {
     }
 
     @Override
-    public IBootstrapSettings setVersion(String version) {
-        this.version = version;
+    public IBootstrapSettings setBootstrapVersion(String version) {
```

Comment:
We cannot break the API in minor releases. Please revert.

Reply:
Done